mne/io/boxy/boxy.py

```diff
@@ -527,6 +533,90 @@ def _read_segment_file(self, data, idx, fi, start, stop, cals, mult):
     
                                 # save our data based on data type
                                 data_[index_loc, :] = boxy_array[:, channel]
+                                
+                    ###phase unwrapping###
+                    # thresh = 0.00000001
+                    # scipy.io.savemat(file_name = r"C:\Users\spork\Desktop\data_matlab.mat",
+                    #                   mdict=dict(data=data_))
+                    if i_data == 'Ph':
+                        print('Fixing phase wrap')
+                        for i_chan in range(np.size(data_, axis=0)):
+                            if np.mean(data_[i_chan,:50]) < 180:
+                                wrapped_points = data_[i_chan, :] > 270
+                                data_[i_chan, wrapped_points] -= 360
+                            else:
+                                wrapped_points = data_[i_chan,:] < 90
+                                data_[i_chan, wrapped_points] += 360
+                                
+                        # unwrapped_data = scipy.io.loadmat(r"C:\Users\spork\Desktop\data_unwrap_python.mat")
+                        
+                        # test1 = abs(unwrapped_data['data'] - data_) <= thresh
+                        # test1.all()
+                                
+                        print('Detrending phase data')
```

Comment:
comment with a description of what this does

Reply:
okay, will do

Reply:
resolve this once done